tree/treeplayer/src/TTreeProcessorMT.cxx

```diff
@@ -416,9 +416,11 @@ std::vector<std::string> TTreeProcessorMT::FindTreeNames()
 /// \param[in] filename Name of the file containing the tree to process.
 /// \param[in] treename Name of the tree to process. If not provided, the implementation will search
 ///            for a TTree key in the file and will use the first one it finds.
-TTreeProcessorMT::TTreeProcessorMT(std::string_view filename, std::string_view treename)
+/// \param[in] nThreads Number of threads to create in the underlying thread-pool. The semantics of this argument are
+///                     the same as for TThreadExecutor.
+TTreeProcessorMT::TTreeProcessorMT(std::string_view filename, std::string_view treename, UInt_t nThreads)
```

Comment:
I don't remember what we converged on, but does it make sense to add a `\note` that if a thread pool already exists, the argument doesn't have any effect?

Reply:
if a thread-pool already exists, and the size is different than the one required, you get a warning. but that's included in "same as for TThreadExecutor", I'd rather point to the one source of truth than duplicate information

Reply:
LGTM